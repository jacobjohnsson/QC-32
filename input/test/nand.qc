LOADIM 0 67108863 /* 25 1's */
LOADIM 1 67108863
NAND 0 0 1 /* r_0 = r_0 NAND r_0 = 0*/
OUTPUT 0 0 0
NAND 0 0 1 /* r_0 = 0 NAND 67108863  = 67108863 */
OUTPUT 0 0 0
HALT
