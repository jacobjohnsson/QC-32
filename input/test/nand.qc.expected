0
33554431
